### PR TITLE
Removed rqScheduler service

### PR DESCRIPTION
**Pull request by nunomsfranco**
- updated python requests
- temp files deleted after use

Commented the delete method in views.py to not break existing implementations.

### CODE REVIEW

File: docsbox/docs/views.py [hunk 1]
```diff
@@ -3,9 +3,9 @@
 
 from flask import request, send_from_directory
 from flask_restful import Resource, abort as flask_abort
-from requests import exceptions
+from requests import exceptions, codes
 from docsbox import app
-from docsbox.docs.tasks import process_convertion, create_tmp_file_and_get_mimetype, get_task, do_task
+from docsbox.docs.tasks import process_convertion, create_tmp_file_and_get_mimetype, get_task, do_task, remove_file
```
Comment (numsu) [inline docsbox/docs/views.py]: Important change?

File: docsbox/docs/views.py [hunk 2]
```diff
@@ -168,20 +172,20 @@ def delete(self, task_id):
...
-                    else:
-                        return 'finished'
-                else:
-                    abort(404, "Unknown tmp_file_remove_task_id", request)
-            else:
-                return 'finished'
-        else:
-            abort(404, "Unknown task_id", request)
\ No newline at end of file
+        # task = get_task(task_id)
```
Comment (numsu) [inline docsbox/docs/views.py]: Should we leave this here to clutter up?

File: docsbox/docs/via_controller.py [hunk 3]
```diff
@@ -5,10 +5,11 @@
...
-    return get(url=url, cert=cert, stream=True)
+    return get(url=url, cert=cert, stream=True, timeout=60)
 
 def save_file_on_via(file_path, mime_type, via_allowed_users):
-    data = open(file_path, "rb")
-    cert=app.config["VIA_CERT_PATH"]
-    headers = {'VIA_ALLOWED_USERS': via_allowed_users, 'Content-type': mime_type}
-    return post(url=app.config["VIA_URL"], cert=cert, data=data, headers=headers)
+    with open(file_path, "rb") as data:
+        cert=app.config["VIA_CERT_PATH"]
```
Comment (numsu) [inline docsbox/docs/via_controller.py]: Fix the spacing between the `=` signs while you're at it. There are several places where they're not aligned

File: docsbox/docs/via_controller.py [hunk 4]
```diff
@@ -5,10 +5,11 @@
...
+        headers = {'VIA_ALLOWED_USERS': via_allowed_users, 'Content-type': mime_type}
```
Comment (numsu) [inline docsbox/docs/via_controller.py]: Also refactor the curly brackets to have spaces inside everywhere

**Review by numsu — APPROVED** [review]

File: docsbox/docs/views.py (hunk 2 above)
Comment (nunomsfranco) [reply docsbox/docs/views.py]: I removed the method since I added version

**Review by martinstf — APPROVED** [review]

**Pull request merged**